continuous-upgrade/jobs/cicd-dump-load-logs-job.yml

```diff
@@ -0,0 +1,17 @@
+- job:
+    name: continuous-upgrade_dump-load-logs-job
+    project-type: freestyle
+    defaults: global
+    builders:
+      - shell: |
+         tail -n 150 /root/svt/reliability/logs/reliability.log
+
+    publishers:
+      - email:
+          recipients: jhadvig@redhat.com skuznets@redhat.com jupierce@redhat.com
+          notify-every-unstable-build: true
```

Comment:
Well wasnt sure if we want to keep it here as it will send the link for the dumped logs, cause we will also get mail if `cicd-start-load-job` on failure, but that wont provide logs.